### DISCUSSION TITLE
consult can't find ripgrep over TRAMP

### DISCUSSION THREAD

**Discussion by JonatanSahar** [minad/consult#1156] [Q&A]
Hi! Sorry to bother you here, but I couldn't find an answer elsewhere..
`consult-ripgrep` works fine locally, but can't find the `rg` executable on the remote machine when running over TRAMP. 
I have the `rg` binary in `~/.local/bin`, and that path is in `$PATH` of my user, appended there in `.bashrc`.  When I manually log in over ssh with the same user defined for TRAMP, `rg` works as expected.
`consult-grep` over TRAMP does work as expected though.

I've manually added the path to  `rg` to `tramp-remote-path`, to no avail. 
Increasing tramp's verbosity gives:

> Tramp: Opening connection  *consult-async-stderr* for yonatan@server using ssh...done
> Tramp: Opening connection rg for yonatan@server using ssh...done
>

**Comment by minad** [comment]
This is a misconfiguration on your server. The bashrc is not necessarily loaded when logging in via Tramp.
